contracts/HtsSystemContract.sol

```diff
@@ -763,7 +763,7 @@ contract HtsSystemContract is IHederaTokenService, IERC20Events, IERC721Events {
 
         // Set the new owner
         assembly { sstore(slot, to) }
-        emit Transfer(from, to, serialId);
+        emit ERC721Transfer(from, to, serialId);
```

Comment:
We can change the contract definition to

```solidity
contract HtsSystemContract is IHederaTokenService {
```

and then emit the events like this

```suggestion
        emit IERC721Events.Transfer(from, to, serialId);
```